app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Online Classroom- <%= @page_title || "" %></title>
+        <style>
+span.status { display: block; width: 10px; height: 10px;
+margin: 0.25em auto; padding: 0; border: 1px solid #000000; } span.status.true { background: green; color: green; } span.status.false { background: red; color: red; }
```

Comment:
Can you try moving your CSS designs to assets/stylesheets?

Reply:
fixed this apu